packages/flet/lib/src/flet_extension.dart

```diff
@@ -12,5 +14,7 @@ abstract class FletExtension {
     return null;
   }
 
-  void ensureInitialized() {}
+  IconData? createIconData(int iconCode) {
```

Comment:
I guess we need to document this a little? And perhaps provide an example on how users should handle the incoming code for their usecases.

Reply:
Right, we will put that in the updated "Extensions guide".